core/src/main/java/io/grpc/internal/ServerStream.java

```diff
@@ -69,7 +69,8 @@
   void cancel(Status status);
 
   /**
-   * Attributes describing stream.
+   * Attributes describing stream.  This is inherited from the transport attributes, and used
+   * as the basis of {@link io.grpc.ServerCall#attributes}.
```

Comment:
"as the basis" makes me think it may not be identical to `ServerCall.attributes()`. Are they not identical?